### PR TITLE
ADS-TEC StoraXe ESS Implementation

### PR DESCRIPTION
**Pull request by greemo**
Hi Stefan,

Here's the ADS-TEC Storage implementation I've been working on.

For the moment it's only a read-only implementation, as that's all we need for now.

ADS-TEC provided me with a simulator for the StoraXe system on which I have tested the implementation.

### CODE REVIEW

**Comment by greemo** [comment]
Lieber Maintainers, eine freundliche Erinnerung. Wir würden gerne dieses Modul verwenden.

**Comment by greemo** [comment]
Hi @sfeilmeier, an update, I have tested this on a live system and it is working. Just waiting for the merge. Cheers.

File: io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java
Comment (sfeilmeier) [inline io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java:120]: This will not work reliably, because somebody could still directly access the `ReactivePower` channel, e.g. for calculation in `_sum`-Component or when visualizing data in InfluxDB/Grafana/OpenEMS UI. You should also use a `ElementToChannelConverter` here.

File: io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java
Comment (sfeilmeier) [inline io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java:126]: You can directly return `GridMode.OFF_GRID` here to make it more obvious; similar below.

File: io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java
Comment (sfeilmeier) [inline io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java:133]: Reverse is not strictly required here, because GridMode can never be set.

**Review by sfeilmeier — CHANGES_REQUESTED** [review]

File: io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java
Comment (greemo) [reply io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java:133]: Yeah, great, I thought we needed to create both. I just saw the implementation for one-way. Thanks!

File: io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java
Comment (greemo) [reply io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java:126]: Thanks!

File: io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java
Comment (greemo) [reply io.openems.edge.ess.adstec.storaxe/src/io/openems/edge/ess/adstec/storaxe/AdstecStoraxeEssImpl.java]: Thanks for this one. I have used a chained converter here.

**Review by sfeilmeier — APPROVED** [review]

**Pull request merged**
